50SAT/SPEX50SAT.bib

```diff
@@ -0,0 +1,11 @@
+@online{PocketQube,
+  title = {{PocketQube} Wikipedia Description},
+  url = {https://en.wikipedia.org/wiki/PocketQube},
+  urldate = {2017-06-18}
+}
+
+@online{50SAT,
```

Comment:
http://www.50dollarsat.info/home.html

Doesn't look like it, or its locked in a journal.